code/__HELPERS/AnimationLibrary.dm

```diff
@@ -0,0 +1,203 @@
+/*
+ * A file intended to store various animation procs for re-use.
+ * A fair majority of these are copy-pasted from Goon and may not function as expected without tweaking.
+ * The spin from being thrown will interrupt most of these animations as will grabs, account for that accordingly.
+ */
+
+/proc/animate_fade_grayscale(var/atom/A, var/time=5)
+	if (!istype(A) && !istype(A, /client))
+		return
+	A.color = list(1,0,0,0, 0,1,0,0, 0,0,1,0, 0,0,0,1)
+	animate(A, color=list(0.33, 0.33, 0.33, 0, 0.33, 0.33, 0.33, 0, 0.33, 0.33, 0.33, 0, 0, 0, 0, 1), time=time, easing=SINE_EASING)
+	return
+
+/proc/animate_melt_pixel(var/atom/A)
+	if (!istype(A))
+		return
+	//A.alpha = 200
+	animate(A, pixel_y = 0, time = 50 - A.pixel_y, alpha = 175, easing = BOUNCE_EASING)
+	animate(alpha = 0, easing = LINEAR_EASING)
+	return
+
+/proc/animate_explode_pixel(var/atom/A)
+	if (!istype(A))
+		return
+	var/floatdegrees = rand(5, 20)
+	var/side = 1
+	side = pick(-1, 1)
+	animate(A, pixel_x = rand(-64, 64), pixel_y = rand(-64, 64), transform = matrix(floatdegrees * (side == 1 ? 1:-1), MATRIX_ROTATE), time = 10, alpha = 0, easing = SINE_EASING)
```

Comment:
`(side == 1 ? 1:-1)`
the `side` assignment should just use `pick(list(-1, 1))` to avoid this `?` operation